docs/reference/elasticsearch/rest-apis/retrievers/linear-retriever.md

```diff
@@ -28,7 +28,7 @@ Combining `query` and `retrievers` is not supported.
     Fields can include boost values using the `^` notation (e.g., `"field^2"`).
     If not specified, uses the index's default fields from the `index.query.default_field` index setting, which is `*` by default.
 
-`normalizer` {applies_to}`stack: ga 9.1`
+`normalizer` {applies_to}`stack: ga 9.2`
```

Comment:
Isn't this misleading that the parameter did not exist (or at least wasn't ga) prior to 9.2?